decode.rb

```diff
@@ -0,0 +1,31 @@
+@morse = '.-', '-...',	'-.-.',	'-..',	'.',	'..-.',	'--.',	'....',	'..',	'.---',	'-.-',
```

Comment:
- [ ] Please create a third function called `decode`, as the project requirement said:
    - `Create a method to decode the entire message in Morse code, take a string parameter, and return the string representation. Every word will be separated by 3 spaces`
    - This method should use `decode_word` function. 🚀